src/app/frontend/error/internalerror_controller.js

```diff
@@ -18,14 +18,18 @@
 export class InternalErrorController {
   /**
    * @param {!./internalerror_state.StateParams} $stateParams
+   * @param {!Object} kdAppConfigService
```

Comment:
It should be possible to provide correct type for this service. Object is too generic.